World.tscn

```diff
@@ -32,6 +160,7 @@ motion_scale = Vector2( 0.2, 0.2 )
 motion_mirroring = Vector2( 4096, 4096 )
 
 [node name="Sprite" type="Sprite" parent="ParallaxBackground/ParallaxLayer1"]
+scale = Vector2( 1.5, 1.5 )
```

Comment:
I'm wondering if changing the scale here will have an impact on the mirroring of the background. 
```suggestion
scale = Vector2( 1, 1 )
```

Reply:
if i left it as the default rigid body you could push it around when you hit it. I think its a cool idea for later on when we get them to move on their own that you could knock them out of their velocity, or just have them weigh a lot so you slow down when you hit them. Setting it to static just makes them act like a static body like the planet is.